UVAtlas/UVAtlas_2017_Win10.vcxproj

```diff
@@ -132,7 +132,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
     <OutDir>Bin\Desktop_2017_Win10\$(Platform)\$(Configuration)\</OutDir>
     <IntDir>Bin\Desktop_2017_Win10\$(Platform)\$(Configuration)\</IntDir>
-    <TargetName>UVAtlas</TargetName>
+    <TargetName>UVAtlasd</TargetName>
```

Comment:
I don't think we normally commit vcxproj?

Reply:
MS originally had all these vs sln files in the repo. This was just to update the naming of the output library. I originally used this before I setup the cmake stuff.